docs/conf.py

```diff
@@ -79,12 +79,14 @@
 from hypothesis import *
 from hypothesis.strategies import *
 # Run deterministically, and don't save examples
+import random
+random.seed(0)
```

Comment:
Could you open an issue about the seeding thing so we don't lose track of it?

Reply:
Opened #966.